### PR TITLE
Update links to RHEL docs (#3429)

### PR DESCRIPTION
**Pull request by rh-max**
#### What changes are you introducing?

Cherrypicking https://github.com/theforeman/foreman-documentation/pull/3429 into 3.10.

If successful, please also cherrypick into versions down to 3.5, thanks.

### CODE REVIEW

**Review by Lennonka — APPROVED** [review]
Awesome, thank you!

**Comment by Lennonka** [comment]
Cherry picked:
- 9264f6ba06..80c9a58344  3.9 -> 3.9 (one conflict resolved)
- f19a6e9c2a..69408ef53c  3.8 -> 3.8 (another conflict resolved)
- a48106de45..0adf6f3fa4  3.7 -> 3.7
- 75af84eda0..4ac80fecaa  3.6 -> 3.6
- 10c7126201..50213b23cf  3.5 -> 3.5

File: guides/common/modules/con_kickstart-workflow.adoc [hunk 1]
```diff
@@ -14,4 +14,4 @@ When you run a {ProjectName} Kickstart script, the script performs the following
 . It runs user defined snippets.
 
 .Additional resources
-For more information about Kickstart, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_8_installation/performing_an_automated_installation_using_kickstart[Performing an automated installation using Kickstart] in _Performing an advanced RHEL 8 installation_.
+For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}8_.
```
Comment (apinnick) [inline guides/common/modules/con_kickstart-workflow.adoc:17]: @Lennonka I just created a [bug](https://issues.redhat.com/browse/SAT-30066) to update these links for RHEL 9. Just wanted to check -- was there any reason for using RHEL 8 links?

Comment (Lennonka) [reply guides/common/modules/con_kickstart-workflow.adoc:17]: If it's about hosts then there should be links to all supported RHEL versions, not only RHEL 8, which is one of the supported client OSs. I hope that helps.

**Pull request merged**